force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -39,18 +39,25 @@ public with sharing class ClassesObjectsMethods {
      * @return A list containing the length of the string, a substring of the string, and the string in lower case.
      */
     public static List<String> practiceStringMethods() {
-        // Initialize a String with 'Hello, Salesforce!'
+                // Initialize a String with 'Hello, Salesforce!'
+                String helloString = 'Hello, Salesforce!';
+                // Get the length of the String        
+                Integer length = helloString.length();
+                String helloLength = string.valueOf(length);
+                // Get a substring of the String of the first 5 characters
+                String helloSubstring = hellostring.substring(0, 5);
+                // Convert the String to lower case
+                String helloLower = helloString.toLowerCase();
+                // Return the list with the string data
+                List<String> stringResult = new List<string>();
+                // In the first position add the string length
+                stringResult.add(helloLength);
+                // In the second position add the substring
+                stringResult.add(helloSubstring);
+                // In the third position add the string in lower case
+                stringResult.add(helloLower);
         
-        // Get the length of the String        
-        // Get a substring of the String of the first 5 characters
-        // Convert the String to lower case
-        
-        // Return the list with the string data
-        // In the first position add the string length
-        // In the second position add the substring
-        // In the third position add the string in lower case
-
-        return null; // Replace null with the variable you used to store the result
+                return stringResult; // Replace null with the variable you used to store the result
```

Comment:
Something I encourage new developers to think about is. "do I really need a variable here" in this case you could the the results of the String method calls directly to the List and, I think, not lose anything in readability.
```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```